apis/tempo/v1alpha1/microservices_types.go

```diff
@@ -104,42 +104,41 @@ type SearchSpec struct {
 	// +optional
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Query timeout"
-	QueryTimeout time.Duration `json:"queryTimeout,omitempty"`
+	QueryTimeout string `json:"queryTimeout,omitempty"`
```

Comment:
Do we avoid the datatype here so that in the end the configuration looks better? Means a value would be `1m5s` instead of `65`?

Reply:
@rubenvp8510 take a look at this PR https://github.com/os-observability/tempo-operator/pull/145

The `time.Duration` can be properly exposed via wrapper `metav1.Duration`.